dv/verilator/simutil_verilator/cpp/verilated_toplevel.h

```diff
@@ -5,18 +5,34 @@
 #ifndef VERILATED_TOPLEVEL_H_
 #define VERILATED_TOPLEVEL_H_
 
+#ifndef TOPLEVEL_NAME
+#error "TOPLEVEL_NAME must be set to the name of the toplevel."
+#endif
+
 #include <verilated.h>
 
+#define STR(s) #s
+
+// Include Verilator-generated toplevel
+#define VERILATED_TOPLEVEL_HEADER_STR2(s) STR(V##s)
+#define VERILATED_TOPLEVEL_HEADER_STR(s) VERILATED_TOPLEVEL_HEADER_STR2(s)
+#include VERILATED_TOPLEVEL_HEADER_STR(TOPLEVEL_NAME.h)
```

Comment:
Please include an empty line before the `#include` so it's obvious you're doing an include - it gets lost in the `#define`s here.